parla/task_runtime.py

```diff
@@ -1653,6 +1669,25 @@ def fill_curr_mapped_task_queue(self):
             if len(new_tasks) > 0:
                 self._mapped_task_queue.extendleft(new_tasks)
 
+    # TODO(lhc): should be refactored by decorator later.
+
+    def update_launched_task_count_mutex(self, task, dev, weight):
+        with self._monitor:
+            if isinstance(task, ComputeTask):
+                self._device_compute_task_counts[dev] += weight
+            else:
+                self._device_datamove_task_counts[dev] += weight
```

Comment:
Why the `weight`?

Reply:
that is just my weird naming.. count might be better name! Thanks!